app/decorators/controllers/frontend/orders_controller/cart_checkout_payment_method_decorator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module CartCheckoutPaymentMethodDecorator
+  def self.prepended(base)
+    base.helper_method :cart_checkout_payment_method
+  end
+
+  # Override this method in order to enable cart page checkout
+  # with Apple Pay and Google Pay. This method must return the
+  # `Spree::PaymentMethod::StripeCreditCard` instance that will
+  # be used for configuring the payment request button, for
+  # example `Spree::PaymentMethod::StripeCreditCard.first`
+  def cart_checkout_payment_method
```

Comment:
I think we might try to remove this file. If we remove deface, we leave to users the responsibility to get this method by suggesting them to use the partial:

```erb
<%= render 'stripe_payment_request_button', cart_checkout_payment_method: Spree::PaymentMethod::StripeCreditCard.first %>
```

This way, it will be explicit that it's up to them to pick the right payment method.